src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+```
+
+__main-scripts.html__
+
+```html
+<script src="main.js"></script>
+```
+
+---
+
+## Using a configuration
+
+You can pass configuration options to ChunksWebpackPlugin to overrides default settings. Allowed values are as follows:
+
+### `outputPath`
+
+`string = null`
+
+Tells the plugin whether to personalize the output path of the generated files.
+
+```js
+new ChunksWebpackPlugin({
+  outputPath: path.resolve(__dirname, './templates')
+});
+```
+
+T> The default behavior will use the  `options.output.path` from the [webpack configuration](https://webpack.js.org/configuration/output/#outputpath).
+
+W> The `outputPath` must be an absolute path.
+
+### `fileExtension`
+
+`string = '.html'`
+
+Tells the plugin whether to personalize the extension of the generated files.
+
+```js
+new ChunksWebpackPlugin({
+  fileExtension: '.html'
+});
+```
+
+### `templateStyle`
+
+`string = '<link rel="stylesheet" href="{{chunk}}" />'`
+
+Tells the plugin whether to personalize the default template for the HTML `<style>` tags. For example, add additional attributes, a CDN prefix or something else.
+
+```js
+new ChunksWebpackPlugin({
+  templateStyle: '<link rel="stylesheet" href="{{chunk}}" />'
+});
+```
+
+T> Keep the `{{chunk}}` placeholder, it is automatically replaced by the concatenation of the webpack public path and the chunk filename.
+
+### `templateScript`
+
+`string = '<script src="{{chunk}}"></script>'`
+
+Tells the plugin whether to personalize the default template for the HTML `<script>` tags. For example, add additional attributes, a CDN prefix or something else.
+
+```js
+new ChunksWebpackPlugin({
+  templateScript: '<script src="{{chunk}}"></script>'
+});
+```
+
+T> Keep the `{{chunk}}` placeholder, it is automatically replaced by the concatenation of the webpack public path and the chunk filename.
+
+### `customFormatTags`
+
+`false || function (chunksSorted, files)`
+
+Tells the plugin whether to personalize the default behavior for generate your own templates. The function is called for each entry points. Custom behavior can also be add for a specific entry point if necessary.
+
+```js
+new ChunksWebpackPlugin({
+  customFormatTags: (chunksSorted, Entrypoint) => {
+    // Generate all HTML style tags with a CDN prefix
+    const styles = chunksSorted.styles.map(chunkCss =>
+      `<link rel="stylesheet" href="https://cdn.domain.com/${chunkCss}" />`
+    ).join('');
+
+    // Generate all HTML style tags with CDN prefix and defer attribute
+    const scripts = chunksSorted.scripts.map(chunkJs =>
+      `<script defer src="https://cdn.domain.com/${chunkJs}"></script>`
+    ).join('');
+
+    return { styles, scripts };
+  }
+});
+```
+
+T> The arrow function syntax allow you to access the class properties.
+
+W> The function provides more flexibility by replacing the default behavior. Follow the example above to make sure it works.
+
+The function must return an object with the following format:
+
+```json
```

Comment:
JavaScript object is not in JSON format unless converted
```suggestion
```json
```

Reply:
True, I updated the doc in this [commit](https://github.com/webpack/webpack.js.org/pull/3771/commits/df67afc4803ebc0732697c768069fa5495e294d0)